docs/nuxt.config.ts

```diff
@@ -36,14 +37,14 @@ export default defineNuxtConfig({
         prefix: '/dev',
         driver: 'fs',
         base: resolve('./content')
-      },
-      // overwrite default source AKA `content` directory
-      content: {
-        driver: 'github',
-        repo: 'nuxt/ui',
-        branch: 'main',
-        dir: 'docs/content'
       }
+      // // overwrite default source AKA `content` directory
```

Comment:
Why did you comment this?

Reply:
China sometimes blocks github, so i disabled github driver in local develop，I apologize that forgot to clean up it.

Reply:
No worries 😊